docs/tutorials/pre_executed/timeseries.ipynb

```diff
@@ -0,0 +1,1471 @@
+{
```

Comment:
<p>I might break the parenthetical section into its own sentence for slightly better readability, ie "...between 1.5 and 9.5. Note that..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/834/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>I like it! Will fix.</p>